bundles/AdminBundle/Controller/Admin/Asset/AssetController.php

```diff
@@ -1023,7 +1023,9 @@ public function saveAction(Request $request, EventDispatcherInterface $eventDisp
                 // save to session only
                 Asset\Service::saveElementToSession($asset);
             } else {
-                $asset->save();
+                $asset->save([
+                    'versionNote' => $request->get('versionNote') ?? null
```

Comment:
```suggestion
                    'versionNote' => $request->get('versionNote')
```
` ?? null` is redundant, as `->get()` returns anyway `null` 🙃